models/iris/iris.sdf.jinja

```diff
@@ -0,0 +1,619 @@
+<sdf version='1.7'>
```

Comment:
We should add a note stating that the gen file was generated from this. Similar to https://github.com/PX4/sitl_gazebo/blob/master/models/matrice_100/matrice_100.sdf.jinja#L95

Reply:
Thanks for pointing that out! added on last commit